docs/commercial/sharing.rst

```diff
@@ -34,6 +34,10 @@ Enabling
    
    You can always revoke access in the same panel.
 
+.. note::
+   
+   You can terminate your session by :ref:`versions:Logging out`.
```

Comment:
Having too many notes feels a bit heavy in the design. I think this should probably just be part of the content.